internal/service/ec2/ec2_launch_template.go

```diff
@@ -1974,9 +1974,10 @@ func expandLaunchTemplateInstanceNetworkInterfaceSpecificationRequest(tfMap map[
 		apiObject.Ipv6Prefixes = expandIPv6PrefixSpecificationRequests(v.List())
 	}
 
-	if v, ok := tfMap["network_card_index"].(int); ok {
-		apiObject.NetworkCardIndex = aws.Int64(int64(v))
-	}
+	// We do not supportNetworkCardIndex
```

Comment:
Может так:
```suggestion
	// CROC Cloud API does not support `NetworkCardIndex` parameter
```

Пробела после `support` точно не хватает